src/SonarQube.Client.Tests/Requests/Api/V7_20/GetIssuesRequestWrapperTests.cs

```diff
@@ -34,10 +35,15 @@ namespace SonarQube.Client.Tests.Requests.Api.V7_20
     [TestClass]
     public class GetIssuesRequestWrapperTests
     {
-        [TestMethod]
-        public async Task InvokeAsync_NoIssueKeys_ExpectedPropertiesArePassedInMultipleRequests()
+        private const string SonarQubeComponentPropertyName = "components";
+        private const string SonarCloudComponentPropertyName = "componentKeys";
+        
+        [DataTestMethod]
+        [DataRow(SonarQubeComponentPropertyName, DisplayName = "SonarQube")]
+        [DataRow(SonarCloudComponentPropertyName, DisplayName = "SonarCloud")]
+        public async Task InvokeAsync_SonarQube_NoIssueKeys_ExpectedPropertiesArePassedInMultipleRequests(string componentPropertyName)
```

Comment:
This UT is for SC and SQ, so SQ likely shouldn't be in the name
```suggestion
        public async Task InvokeAsync_NoIssueKeys_ExpectedPropertiesArePassedInMultipleRequests(string componentPropertyName)
```